rivet.spec

```diff
@@ -11,44 +11,26 @@ Patch0: rivet-1.4.0
 %endif
 
 %if "%{?cms_cxxflags:set}" != "set"
-%define cms_cxxflags -O2 -std=c++11
+%define cms_cxxflags -O2 -std=c++0x
```

Comment:
Why `c++0x`? Technically it's fine even if you set it to `c++14` as that is what we use for CMSSW.